packages/frontend/src/services/indexer/cached-api.js

```diff
@@ -0,0 +1,26 @@
+import api from './api';
+import Cache from './cache';
+
+
+const UPDATE_REQUEST_INTERVAL = 1000 * 30;
+const LIKELY_NFT_KEY = 'likelyNFTs';
+const LIKELY_TOKENS_KEY = 'likelyTokens';
+
+export default {
+    listLikelyNfts(accountId) {
+        return Cache.accumulate({
+            accountId,
+            kind: LIKELY_NFT_KEY,
+            updater: (timestamp) => api.listLikelyNfts(accountId, timestamp),
+            timeout: UPDATE_REQUEST_INTERVAL,
+        });
+    },
+    listLikelyTokens(accountId, timestamp) {
+        return Cache.accumulate({
+            accountId,
+            kind: LIKELY_TOKENS_KEY,
+            updater: (timestamp) => api.listLikelyTokens(accountId, timestamp),
+            timeout: UPDATE_REQUEST_INTERVAL,
```

Comment:
Is the intended behavior to call `listLikelyTokens` endpoint every 30 seconds?

I was looking over network requests while running the app locally and I did not really see that happen. Messed around with `UPDATE_REQUEST_INTERVAL` and it still did not work. Am I missing something?

Reply:
It is a bug, thank you.

Reply:
Current time should be converted to nanoseconds before comparison, fixed.
lastTime from backend should be parsed as integer.

Fixed.